DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/EnrolController.cs

```diff
@@ -255,17 +255,15 @@ public IActionResult EnrolDelegateSummary(int delegateId, string delegateName)
         [HttpPost]
         public IActionResult EnrolDelegateSummary(int delegateId)
         {
+            var delegateUserId = User.GetUserIdKnownNotNull();
+            var centreId = User.GetCentreIdKnownNotNull();
             var clockUtility = new ClockUtility();
             var sessionEnrol = multiPageFormService.GetMultiPageFormData<SessionEnrolDelegate>(
                MultiPageFormDataFeature.EnrolDelegateInActivity,
                TempData);
             if (!sessionEnrol.IsSelfAssessment)
             {
-                progressDataService.CreateNewDelegateProgress(delegateId, sessionEnrol.AssessmentID.GetValueOrDefault(), sessionEnrol.AssessmentVersion,
-                  clockUtility.UtcNow, 0, GetAdminID(), sessionEnrol.CompleteByDate, sessionEnrol.SupervisorID.GetValueOrDefault());
-
                 enrolService.EnrolDelegateOnCourse(delegateId, sessionEnrol.AssessmentID.GetValueOrDefault(), sessionEnrol.AssessmentVersion, 0, GetAdminID(), sessionEnrol.CompleteByDate, sessionEnrol.SupervisorID.GetValueOrDefault(), "AdminEnrolDelegateOnCourse");
```

Comment:
Was this the duplicate call? Just curious! (Deleting existing code makes me anxious lol).